README.md

```diff
@@ -90,6 +91,17 @@ Options can be combined:
 ./build.sh run --scale 7.0 --seed 42 'abstract art'
 ```
 
+On systems with <8GB of GPU RAM, you can try mixing and matching options:
+
+* Make images smaller than 512x512 using `--W` and `--H`
+* Use `--half` to decrease memory use but slightly decrease image quality
+* Use `--attention-slicing` to decrease memory use but decrease image speed
```

Comment:
```suggestion
* Use `--attention-slicing` to decrease memory use but also decrease image generation speed
```

Reply:
I used the word `creation` since `create` is used in the rest of the README.